conda-recipes/colorlog-feedstock/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Can you move this file into the `/recipe` folder and then reference it in the `about` section of the meta.yaml file?
Also, we'll want a LICENSE file for the recipe code as well. It should be BSD or apache-2 licensed.

Reply:
OK, only this issue left. Generally we want:
```
my-feedstock/
my-feedstock/LICENSE            (this one is for the recipe itself)
myfeedstock/recipe/LICENSE   (this is a copy of the license the content uses)
```

Reply:
Ok I added a BSD LICENSE file inside the recipe folder.